README.md

```diff
@@ -0,0 +1,11 @@
+This small script continuously builds a LaTeX file in the background and outputs all files into a folder of your choice (default `output/`). Of note is the build file (default `output/build.pdf`) which is what you should open in a pdf viewer of your choice while editing your LaTeX document.
+
+It is less jarring and looks cleaner than what I used to use (`latexmk -pvc`) (The final build file only updates on a successful and complete build. Other build tools will write an incomplete pdf that your pdf viewer can't handle, leading to a broken pdf for a few seconds while you are editing) I also think it's faster, but that might just be the placebo effect.
+
+See [the original stack overflow post](http://stackoverflow.com/questions/1240037/recommended-build-system-for-latex/1394702#1394702) for motivation, usage instruction, and some spicy vim configuration.
+
+Note, this repository is a fork. Changes:
```

Comment:
I think this is left in from the fork?

Reply:
Good point, changed the wording somewhat. Let me know if it's still too unprofessional.